### ISSUE TITLE
Test email not sent with SMTP

### ISSUE THREAD

**Issue by Mutusen** [mettle/sendportal#301]
Hi, I'm testing Sendportal because it seems promising, but I just can't sent test emails. I have a mail server hosted on the same server as Sendportal, I set SMTP options as I always do, I send a test email... and the nothing. No email is sent, there is nothing in the email server logs and in Apache error logs. I tried to set an incorrect SMTP password to see if there's an error message, and there isn't, the only thing I see is "The test email has been dispatched."

Did anyone encounter a similar problem? I'm using the latest version (3.0.1).

**Comment by Farazdev7** [comment]
I also just installed sendPortal after banging my head for 3 days and now its not sending the message.
I am using SendPortal v2.
First I tried to connect it to Amazon SES, it did not worked, now I tried to connect it with AWS SES SMTP and its showing 
"The test email has been dispatched."
but no message gets send.

**Comment by BenMenking** [comment]
Per @Farazdev7  issue, SES needs to be setup correctly for identities.

@Mutusen do you have any error information?  It could be AWS permissions, SES configuration or a host of other issues.

**Comment by Mutusen** [comment]
@BenMenking No, as I said there is no error message anywhere (either in email logs or Apache logs). There's no AWS or SES, the email server is postfix running on the same server as Sendportal.

**Comment by BenMenking** [comment]
That's what I get for not reading the OP :-)

What about sending to an external SMTP server, does that work?  If SendPortal were actually sending the test email I'd expect activity in /var/log/maillog (or whatever log Postfix is writing to locally).
